.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+# As opposed to https://github.com/kubernetes-sigs/cluster-api (CAPI), the CAPA upstream project does not offer
```

Comment:
There are now two files `.github/workflows/release.yaml` and `.github/workflows/release.yml`. Let's keep only the `.yaml` file extension which upstream uses.

Reply:
Fixed.